src/halmos/__main__.py

```diff
@@ -433,12 +455,12 @@ def run(
             elif args.print_potential_counterexample:
                 print(color_warn(f'Counterexample (potentially invalid): {str_model(model, args)}'))
             else:
-                print(color_warn(f'Counterexample: unknown'))
-        else:
-            print(color_warn(f'Counterexample: unknown'))
+                print(color_warn(f'Counterexample: {result}'))
```

Comment:
this doesn't print anything when model is None and result is unknown.  is it intended?

Reply:
no, we probably want to warn